packages/dd-trace/src/config.js

```diff
@@ -842,7 +842,6 @@ class Config {
     tagger.add(tags, options.tags)
 
     this._setBoolean(opts, 'appsec.apiSecurity.enabled', options.appsec.apiSecurity?.enabled)
-    this._setUnit(opts, 'appsec.apiSecurity.requestSampling', options.appsec.apiSecurity?.requestSampling)
```

Comment:
According to the RFC there is no programmatic option for this, just an env var

Reply:
Yeah the RFC rarely mention those as they're language dependant, it's really up to us to decide if we want to make them available or not. I'd say why not ?

Reply:
AFAIK backend set the 30s window for better accuracy. It’s mainly for backend use, not user configuration.
the env var is just for testing, and changing it could be risky